acai-ios/ViewControllers/MenuViewController.swift

```diff
@@ -26,10 +26,10 @@ class MenuViewController: UIViewController {
     // MARK: Lifecycle
     override func viewDidLoad() {
         super.viewDidLoad()
-        
+
         view.backgroundColor = .white
 
-        title = "Good Morning, Jamie ☀️"
+        title = "Good Morning, Cathy ☀️"
```

Comment:
This might cause some merge conflicts with my menu stuff so you might want to change that back.